app/views/admin/conversations/_table.html.haml

```diff
@@ -31,11 +31,11 @@
           %td= link_to conversation.ride.name, admin_ride_path(conversation.ride)
         - else
           %td Not yet
-        %td= link_to 'Close', close_admin_conversation_path(conversation), class: 'btn btn-default btn-xs', data: { confirm: 'Are you sure?' }, method: :POST
+        %td= link_to 'Close', close_admin_conversation_path(conversation), class: 'btn btn-warning btn-xs', data: { confirm: 'Are you sure? Permanently closes the conversation and ride, removes rider from the pool.' }, method: :POST
```

Comment:
I don't know whether there's a style & voice guideline somewhere, but this isn't a sentence. It's more of a comma-splice-fragment. A semicolon would better separate the two fragments:

> Permanently closes the conversation and ride; removes the rider from the pool.

Even better, you could make it a sentence:

> This will permanently close the conversation and ride and remove the rider from the pool.

Reply:
Thanks, I made it a sentence.